Sources/BarcodeScannerController.swift

```diff
@@ -320,8 +324,25 @@ open class BarcodeScannerController: UIViewController {
 
   // MARK: - Layout
   func setupFrame() {
-    headerView.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: 64)
-    flashButton.frame = CGRect(x: view.frame.width - 50, y: 73, width: 37, height: 37)
+    let flashButtonSize: CGFloat = 37
+    let isLandscape = view.frame.width > view.frame.height
+    let isIphoneX = UIDevice().userInterfaceIdiom == .phone && UIScreen.main.nativeBounds.height == 2436
```

Comment:
The layout shouldn't rely on `UIScreen`, it should depends on the view of the view controller instead